wagtail/admin/templates/wagtailadmin/reports/locked_pages.html

```diff
@@ -1,5 +1,5 @@
 {% extends 'wagtailadmin/reports/base_page_report.html' %}
-{% load i18n %}
+{% load i18n wagtailadmin_tags %}
```

Comment:
I guess this isn't needed?

Reply:
checked it again. not needed. dropped and rebased